src/styles/theme/oc-checkbox.scss

```diff
@@ -3,10 +3,68 @@
 //
 // ========================================================================
 
+@mixin oc-form-check-size($factor) {
+  height: $form-check-size-default * $factor;
+  width: $form-check-size-default * $factor;
+}
+
 .oc-checkbox {
-  @extend .uk-checkbox;
+  @include oc-form-check-size(1);
+
+  display: inline-block;
+  overflow: hidden;
+  vertical-align: middle;
+  -webkit-appearance: none;
```

Comment:
Codacy found an issue: [Expected -webkit-appearance to come before vertical-align (order/properties-alphabetical-order)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6255785)